rmw_zenoh_cpp/src/impl/client_impl.hpp

```diff
@@ -33,13 +33,17 @@ struct rmw_client_data_t
   // Request-response sequence id (To identify and match individual requests)
   static std::atomic<std::int64_t> sequence_id_counter;
 
+  // *INDENT-OFF* because uncrustify can't decide which way to format this
```

Comment:
Likewise confused with the indentation stuff here

Reply:
Same as https://github.com/osrf/rmw_zenoh/pull/26#discussion_r486080963